node.js

```diff
@@ -24,6 +24,8 @@ const chord = grpc.loadPackageDefinition(packageDefinition).chord;
 const userMap = {};
 const HASH_BIT_LENGTH = 3;
 const CHECK_NODE_TIMEOUT_ms = 1000;
+const DEFAULT_HOST_NAME = "localhost";
```

Comment:
These are nice. What do you think about using hostName in place of ip throughout our codebase? If you like the idea, we should open an issue to refactor.

Reply:
Agreed. See issue #63